frontend/src/App.js

```diff
@@ -1,20 +1,24 @@
-import { useRoutes } from 'react-router-dom';
-import { SnackbarProvider } from 'notistack';
 import { ThemeProvider } from '@mui/material';
-import GlobalStyles from './components/GlobalStyles';
-import SplashScreen from './components/SplashScreen';
-import useAuth from './hooks/useAuth';
-import useScrollReset from './hooks/useScrollReset';
-import useSettings from './hooks/useSettings';
+import { SnackbarProvider } from 'notistack';
+import { useRoutes } from 'react-router-dom';
+import { useAuth } from './authentication';
+import {
+  GlobalStyles,
+  SplashScreen,
+  createMaterialTheme,
+  useScrollReset,
+  useSettings
+} from './design';
 import routes from './routes';
-import { createMaterialTheme } from './theme';
 
-const App = () => {
+export const App = () => {
   const content = useRoutes(routes);
   const { settings } = useSettings();
   const auth = useAuth();
   useScrollReset();
 
+  console.log('hello');
```

Comment:
Should be removed

Reply:
Yes, my bad 😁

Reply:
I updated the console rule to not allow any `console.log` only `error` and `info` as they are used in some places meaningfully.